aip/general/0203.md

```diff
@@ -24,10 +24,12 @@ field behavior, such as a field being required or immutable.
 RecognitionAudio audio = 2 [(google.api.field_behavior) = REQUIRED];
 ```
 
-- APIs **must** apply the `google.api.field_behavior` annotation on every field.
+- APIs **must** apply the `google.api.field_behavior` annotation on every field
```

Comment:
When did this become a requirement? It shouldn't be one. Leaving off a `field_behavior` annotation is equivalent to specifying `OPTIONAL`.

Reply:
See https://github.com/aip-dev/google.aip.dev/pull/1100, there's a separate thread you opened we could discuss too.